autoprotocol/protocol.py

```diff
@@ -4979,7 +4973,7 @@ def provision(self, resource_id, dests, volumes):
                     d.container):
                 self.instructions[-1].to.append(xfer)
```

Comment:
I think the issue with the lint warning is on this branch.. here, we're just appending without returning.

We probably want to return the modified instruction here (i.e. self.instructions[-1]). Generally the modify in-place isn't ideal.. but I think returning what we modified would make sense

Reply:
Now that we are returning provision object, I think this comment is addressed

Reply:
yup! let's update the return statement, but this looks good otherwise